packages/deb/hazelcast/DEBIAN/conffiles

```diff
@@ -1,7 +1 @@
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast-client-failover-full-example.xml
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast-client-failover-full-example.yaml
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast-client-full-example.xml
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast-client-full-example.yaml
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast-full-example.xml
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast-full-example.yaml
-/usr/lib/hazelcast/hazelcast-4.1-BETA-1/bin/hazelcast.xml
+
```

Comment:
@fbarotov, what is the reason for removing these?

Reply:
In Hazelcast 5.0 distribution these xml/yaml files will not be under `bin/` directory. There is a PR on consolidating all configuration related files under `examples/` directory: https://github.com/hazelcast/hazelcast/issues/18990

After that PR is merged we can apply updates here as well.

Reply:
@fbarotov these files are considered as user config files and are marked in `conffiles` to prevent overwriting when a new version is installed. A similar marking exists in the `hazelcast.spec` file as `%config(noreplace)` keyword for RPM packages.

Reply:
Ohh, I see, thank you for the explanation